lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -77,34 +85,20 @@ def self.keywords
         rule %r/\s+/, Text::Whitespace
         rule %r/\)/, Punctuation, :pop!
         rule %r/\(/, Punctuation
-        rule %r/[a-z0-9\/_.]*/i, Str
+        rule %r/[a-z0-9\/_.-]*/i, Str
       end
 
       state :strings do
         rule %r/(\([a-z0-9_]+\))?[-#0 +]*([0-9]+|[*])?(\.([0-9]+|[*]))?/i, Str
       end
 
       state :strings_double do
-        rule %r/[^\\"%\n]+/, Str
+        rule %r/[^"%\n]+/, Str
         mixin :strings
       end
 
-      state :escape do
-        rule %r(\\
-          ( [\\abfnrtv"']
-          | \n
-          | N{[a-zA-z][a-zA-Z ]+[a-zA-Z]}
-          | u[a-fA-F0-9]{4}
-          | U[a-fA-F0-9]{8}
-          | x[a-fA-F0-9]{2}
-          | [0-7]{1,3}
-          )
-        )x, Str::Escape
-      end
-
       state :dqs do
         rule %r/"/, Str, :pop!
-        mixin :escape
         mixin :strings_double
       end
```

Comment:
I don't remember now why the states were broken out like this but since `:strings_double` is never used by anything else, it seems like you could consolidate all the double-quoted string rules into a single state.

Reply:
I've done this.